fly_video_assignment_away/__main__.py

```
@@ -0,0 +1,223 @@
+                "i/o operation on closed pipe",
+                "closed pipe",
+                "unclosed transport",
+            ]
+        ):
+            return  # 静默忽略
+    # 其他异常使用默认处理
+    sys.__unraisablehook__(unraisable)
+
+
+def _custom_excepthook(exc_type, exc_value, exc_traceback):
+    """全局异常处理器，将未捕获的异常记录到日志文件"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)